README.md

```diff
@@ -26,22 +29,31 @@ like to host it locally on your computer, you need to do the following:
    folder of this repository.
 5. Run `bundle exec jekyll serve` to start the website.
 6. Open `http://localhost:4000` in a browser.
-7. In Visual Studio Code, install the following plugins:
-   * `davidanson.vscode-markdownlint`, to lint Markdown files according to our
-     defined set of rules.
-   * `shd101wyy.markdown-preview-enhanced`, to render Markdown to HTML in a
-     preview window.
-   * `bpruitt-goddard.mermaid-markdown-syntax-highlighting`, to give syntax
-     highlighting to Mermaid diagrams in Markdown files.
-   * `yzhang.markdown-all-in-one`, to enable a plethora of Markdown features,
-     most importantly formatting of Markdown tables with VS Code's built-in
-     format functionality.
-   * `stkb.rewrap`, to make line-breaking text at 80 characters easier.
-   * `supperchong.pretty-json` to format selected JSON snippets in code
-     examples.
-   * `sissel.shopify-liquid` for syntax highlighting of [Liquid][liquid].
-8. Also in Visual Studio Code, [set up a ruler at 80 characters][vsc-ruler]
-   by adding `"editor.rulers": [80]` to its configuration.
+
+### Docker
+
+1. Install [Docker][docker], using Linux containers.
+   Make sure virtualization is enabled on your machine.

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5367554)

Reply:
Why would you want a newline after list item?